build.sh

```diff
@@ -33,7 +33,7 @@ case "$ARCH" in
 	'aarch64'|'alpine-aarch64')
 		export GOARCH=arm64
 	;;
-	'alpine-armhf')
+	'alpine-armv6hf'|'alpine-armv7hf')
```

Comment:
@nghiant2710  Does `armv6hf` not need GOARM=6 
https://github.com/golang/go/wiki/GoArm#supported-architectures

Reply:
@imrehg I did some tests with binary built with `GOARM=6` and it works so it's better to change as you mentioned. I updated the PR as well